### PR TITLE
Adds shark recipe

### PR DESCRIPTION
**Pull request by mpre**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by mpre** [comment]
@bioconda/core Hi, I need to add the `please review & merge` label to this pull request but I'm not a member of bioconda.  Could you please add me?

**Comment by ldenti** [comment]
@bioconda/core mpre cannot ping you

File: recipes/shark/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - {{ compiler('cxx') }}
+
+  host:
+    - zlib
+    - sdsl-lite
+    - tbb
+    - tbb-devel
+
+  run:
+    - tbb
```
Comment (dpryan79) [inline recipes/shark/meta.yaml]: Can you add `zlib` here too?

File: recipes/shark/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+test:
+  commands:
+    - shark -h
+
+about:
+  home: https://algolab.github.io/shark/
+  license: GPL-3.0-or-later
+  summary: Mapping-free filtering of useless RNA-Seq reads
```
Comment (dpryan79) [inline recipes/shark/meta.yaml]: Please add `license_file: LICENSE`

**Comment by dpryan79** [comment]
Thanks @mpre, you should have now received your invitation to join bioconda.

File: recipes/shark/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  run:
+    - tbb
+
+test:
+  commands:
+    - shark -h
+
+about:
+  home: https://algolab.github.io/shark/
+  license: GPL-3.0-or-later
```
Comment (bgruening) [inline recipes/shark/meta.yaml]: please include a `license_file`

Comment (dpryan79) [reply recipes/shark/meta.yaml]: They did :)

**Pull request merged**